### ISSUE TITLE
V3: Geo

### ISSUE THREAD

**Issue by telephone** [bcosca/fatfree#181]
On commit 11b9c4e you made the array key `region` to be passed with geoip_\* (you accidently left `region` out of geoPlugin).

But that's not the issue. In the pull request I left out the `region` key as it conflicts between geoip_ and geoPlugin:

geoip_ returns `region` as the region code, where as geoPlugin returns `region` as the region name.

Hence why I unset/blocked the `region` key. Along with the fact that the values are passed via `region_code` and `region_name`.

**tl;dr:** To unify array output, `region` key needs be removed
